README.md

```diff
@@ -334,6 +335,16 @@ Params:
 @return stock.location (as described above, containing the quants in the format also listed above).
 ```
 
+### Stock Location PI Count
+```
+URI: /api/stock-location-pi-count/
+Method: POST
+Params:
+@param pi_request: JSON object with "pi_count_moves", "inventory_adjustments", "preceding_inventory_adjustments" and "location_id" entries
+```
+
```

Comment:
Can we add a brief description of what it is going to do?

Reply:
Done.